### PR TITLE
[Lang] Add errror message for missing init call

### PR DESCRIPTION
**Pull request by neozhaoliang**

### CODE REVIEW

File: python/taichi/lang/impl.py [hunk 1]
```diff
@@ -490,7 +490,13 @@ def create_field_member(dtype, name):
     dtype = cook_dtype(dtype)
 
     # primal
-    x = Expr(get_runtime().prog.make_id_expr(""))
+    try:
+        x = Expr(get_runtime().prog.make_id_expr(""))
+    except:
+        raise TaichiRuntimeError(
```
Comment (ailzhang) [inline python/taichi/lang/impl.py:496]: This fix is a bit hacky, would you mind explaining  which issue are you trying to solve and what was the original error message?

Comment (ailzhang) [reply python/taichi/lang/impl.py:496]: Yea then the right way is to check if `get_runtime().prog is None` instead of this `try...except` block?

Comment (neozhaoliang) [reply python/taichi/lang/impl.py:496]: Yes that's a better solution.

**Review by ailzhang — CHANGES_REQUESTED** [review]
Would you mind splitting these to two PRs? Thanks!
(I feel like the change to `floor` is a bit tricky and this PR tries to fix two things in one place.

**Comment by neozhaoliang** [comment]
> Would you mind splitting these to two PRs? Thanks! (I feel like the change to `floor` is a bit tricky and this PR tries to fix two things in one place.

Sure, it 's now split into two PRS.

**Review by ailzhang — APPROVED** [review]
LGTM! 
(Btw you'll have to fix pylint errors manually ;)

**Comment by neozhaoliang** [comment]
Fixed.

**Pull request merged**